packages/@expo/fingerprint/src/sourcer/Expo.ts

```diff
@@ -199,12 +200,26 @@ export async function getExpoAutolinkingIosSourcesAsync(
     return [];
   }
 
+  let platformParam = 'apple';
+  const expoPackageRoot = resolveFrom.silent(projectRoot, 'expo/package.json');
```

Comment:
Can't we skip checking the `expo` package and only resolve `expo-modules-autolinking` from the project root?

Reply:
that would break pnpm isolated mode because projectRoot does not have a direct dependency to expo-modules-autolinking.